.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,19 @@
+## Source: https://github.com/marketplace/actions/ansible-lint
+
+name: Ansible Lint # feel free to pick your own name
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Important: This sets up your GITHUB_WORKSPACE environment variable
+      - uses: actions/checkout@v2
```

Comment:
Latest version of checkout is v3.

Reply:
Good catch, I just copied default config from ansible-lint workflow.

Reply:
Done